js/src/VueRenderer.js

```diff
@@ -77,8 +77,11 @@ function pickSerializable(object, depth=0, max_depth=2) {
 export function eventToObject(event) {
     if (event == null) {
         return event;
+    } else if(event instanceof Event) {
+        return pickSerializable(event);
+    } else {
```

Comment:
nit-pick, but `else` can also be left out

Reply:
you mean you prefer
```javascript
    if(event instanceof Event) {
        return pickSerializable(event);
    }
    return event;
```
over
```
    if(event instanceof Event) {
        return pickSerializable(event);
    } else {
        return event;
    }
```
right?
I do too, and I do this in Python by default, but I'm not sure it is idiomatic JS.

Reply:
yeah, I prefer the first one. I never thought of that being language dependent.